builder/azure/chroot/builder.go

```diff
@@ -0,0 +1,452 @@
+//go:generate struct-markdown
+
+// Package chroot is able to create an Azure manage image without requiring the
+// launch of a new instance for every build. It does this by attaching and
```

Comment:
_> launch of a new instance for every build_
Instance of what, a VM?

Reply:
You're right, 'instance' is not really Azure language. Changed it to 'virtual machine'.